client/lib/desktop/index.js

```diff
@@ -104,14 +102,14 @@ const Desktop = {
 	sendUserLoginStatus: function () {
 		let status = true;
 
-		if ( user.data === false || user.data instanceof Array ) {
+		if ( user().data === false || user().data instanceof Array ) {
```

Comment:
The check for `instanceof Array` is obsolete and can be removed. Before  #36781, the `User.prototype.clear` was setting `this.data = []` (I don't know why), but today it sets it to `false`.

Reply:
Done! Also switched from `data` to `get()`.